frontend/src/components/header/index.jsx

```diff
@@ -0,0 +1,64 @@
+import { For } from "solid-js";
+import { styled } from "solid-styled-components";
+import NavLink from "./nav-link";
+
+const Header = (props) => {
+  return (
+    <Wrapper>
+      <Navigation>
+        <For each={props.navLinks}>
+          {({ name, href }) => <NavLink name={name} href={href} />}
```

Comment:
Do you preserve reactivity here when unpacking an object?

Reply:
theese component not tracked that's why i think it is ok 
https://www.solidjs.com/tutorial/flow_for

Reply:
What do you mean by 'tracked'?